CONTRIBUTING.md

```diff
@@ -42,27 +52,21 @@ If your example uses multiple services and you aren't sure where to add it to th
 create an issue and describe what your code example does. 
 One of the AWS SDK code example team members will follow up with you in that issue.
 
-## How to contribute
-
 To contribute, send us a pull request. For small changes, such as fixing a typo or adding a link, you can use the [GitHub Edit Button](https://blog.github.com/2011-04-26-forking-with-the-edit-button/). For larger changes:
 
 1. [Fork the repository](https://help.github.com/articles/fork-a-repo/).
 2. In your fork, make your change in a branch that's based on this repo's **main** branch.
 3. Commit the change to your fork, using a clear and descriptive commit message.
 4. [Create a pull request](https://help.github.com/articles/creating-a-pull-request-from-a-fork/), answering any questions in the pull request form.
 
-Before you send us a pull request, please be sure that:
+### Before you send us a pull request, please be sure that:
 
 1. You're working from the latest source on the **main** branch.
 2. You check [existing open](https://github.com/awsdocs/aws-doc-sdk-examples/pulls), and [recently closed](https://github.com/awsdocs/aws-doc-sdk-examples/pulls?q=is%3Apr+is%3Aclosed), pull requests to be sure that someone else hasn't already addressed the problem.
 3. You [create an issue](https://github.com/awsdocs/aws-doc-sdk-examples/issues/new) before working on a contribution that will take a significant amount of your time.
```

Comment:
I feel this is buried. We talk about pull requests above without mentioning issues. Maybe move the who section to before the previous one? (as a sub-heading ###)